src/org/zalando/stups/kio/api.clj

```diff
@@ -84,10 +85,19 @@
 
 ;; applications
 
+;;https://github.com/clojure/core.memoize/blob/master/docs/Using.md#overriding-the-cache-keys
+(defn ^{:clojure.core.memoize/args-fn #(merge (first %) (-> (second %) meta :name))}
```

Comment:
Is `(-> (second %) meta :name)` a dictionary? How does merge work if it's nil or a string?